config/locales/en.yml

```diff
@@ -380,4 +381,4 @@ en:
     title: 'Unsubscribe From Our Mailing List'
     unsubscribe: 'Unsubscribe'
     success: You have successfully unsubscribed from our mailing list.
-    invalid_token: Invalid subscription token
+invalid_token: Invalid subscription token
```

Comment:
@sambhav2612 Can you revert this whitespace change as it's breaking the tests

Reply:
it's done